docs/CONTRIBUTING.md

```diff
@@ -29,7 +29,8 @@ An easier solution would be to use Docker.
 **Setup**
```

Comment:
What if we breakdown this section to two or three parts:
- Using npm to install dependencies and have Redis installed
- Have Docker installed and just use 'docker-compose up --build' command (temporary *)
- Optionally suggesting a mix of two, for example, npm to install dependencies and Redis from Docker image (as it is in your pull request)

*Our Docker files for development and debug environments are coming by @wajeehsheikh .

Reply:
The way @PagerDuty is suggesting doesn't need Redis installed on your machine, that's why I think it's the best option. Running `docker-compose up -d redis` is enough. If anything, we should remove the `Redis set up` and just add instructions to install `docker`.

Reply:
That't true. 
```
Optionally suggesting a mix of two, for example, npm to install dependencies and 
{ use } Redis from Docker image (as it is in your pull request)
```

I have missed "{ use }" in this 👆sentence.